contracts/abstracts/ListingManager.sol

```diff
@@ -43,6 +44,14 @@ abstract contract ListingManager is Controlable {
     emit ListingCreated(listingId, tokenContract, tokenId, salePrice, seller);
   }
 
+  function _editListingPrice(uint256 listingId, uint256 newSalePrice) internal returns (bool success) {
+    require(_listings[listingId].salePrice == 0, 'NFT minted');
```

Comment:
This is not true, we never erase the sale price, so the only way salePrice == 0 is if the listingId is not used
And I'm not sure what you mean by 'NFT minted'? This contract does not mint nfts...